spartan/metrics/terraform/grafana.tf

```diff
@@ -0,0 +1,127 @@
+# See https://registry.terraform.io/providers/grafana/grafana/latest/docs
+
+terraform {
+  required_providers {
+    grafana = {
+      source  = "grafana/grafana"
+      version = "~> 3.13.2"
+    }
+  }
+}
+
+provider "grafana" {
+  url  = var.grafana_url
+  auth = var.grafana_auth
+}
+
+resource "grafana_folder" "rule_folder" {
+  title = "Alerting Rules"
+}
+
+
+resource "grafana_contact_point" "slack" {
+  name = "slack"
+
+  slack {
+    url = var.slack_url
+  }
+}
+
+resource "grafana_notification_policy" "ignore_policy" {
+  contact_point = grafana_contact_point.slack.name
+  group_by      = ["service_namespace"]
+
+  policy {
+    contact_point = grafana_contact_point.slack.name
+
+    matcher {
+      label = "service_namespace"
+      match = "="
+      value = "smoke"
+    }
+
+    mute_timings = ["always"]
+  }
+}
+
+resource "grafana_mute_timing" "mute_timing_always" {
+  name = "always"
+
+  intervals {
```

Comment:
Does this read as muting always? should some interval be in here?

Reply:
Yes, the only notification policy that uses this "mute always" is when the `service_namespace` is `smoke`. Thoughts?